avocado/utils/linux.py

```diff
@@ -60,3 +60,17 @@ def is_selinux_enforcing():
     if '1' in genio.read_one_line('/sys/fs/selinux/enforce'):
         return True
     return False
+
+
+def enable_selinux_enforcing():
+    """
+    Enable  SELinux Enforcing in system
+
+    :return: True if SELinux enable in enforcing mode, False if not enabled
+    """
+    if is_selinux_enforcing():
+        return True
+    genio.write_one_line('/sys/fs/selinux/enforce', '1')
```

Comment:
Since the kernel will handle properly to write or not, I would suggest removing the lines 71-72.

Reply:
ok sure , make sense will do